### ISSUE TITLE
Store trusted authorities for Verifiable Credential issuance

### ISSUE THREAD

**Issue by zonotope** [fluree/db#284]
Store trusted authorities for issuing verifiable credentials granting privileges to query or commit to a ledger within the ledger, and look up these authorities to validate credentials presented in client requests

**Comment by bplatz** [comment]
@mpoffald I don't think this is alpha 1 - it would touch the policy work and build on top of it, but brings with it an issue of support for many did methods to be useful - something we don't want in Fluree directly (but would support some method for Fluree to delegate that to an independent service).

This needs to be designed and will end up with I expect multiple tickets here, and a new repo to build a separate service that does this work.
